lib/extensions/response.extensions.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:http/io_client.dart';
```

Comment:
Before I've had conflicts around supporting Flutter Web because of imports like this, so I'm writing a mental note to make sure I check it later on